help_to_heat/frontdoor/migrations/0008_auto_20230721_1456.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.19 on 2023-07-21 14:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("frontdoor", "0007_answer_frontdoor_a_session_cfe892_idx"),
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name="answer",
+            name="frontdoor_a_session_cfe892_idx",
+        ),
+        migrations.AddIndex(
+            model_name="answer",
+            index=models.Index(
+                fields=["session_id", "page_name", "-created_at"],
```

Comment:
Is the `-` on `created_at` intended?

Reply:
yeah, it's descending, because we are getting the latest one in a query, but never the first, so a small optimization

Reply:
Ahh, gotcha. I was curious what the extra syntax did 😄 Nice.